diff --git a/.DS_Store b/.DS_Store
index 3a917a0..84bd3b9 100644
Binary files a/.DS_Store and b/.DS_Store differ
diff --git a/Creator AI Studio.xcodeproj/project.pbxproj b/Creator AI Studio.xcodeproj/project.pbxproj
index aa28ecf..2fb798f 100644
--- a/Creator AI Studio.xcodeproj/project.pbxproj	
+++ b/Creator AI Studio.xcodeproj/project.pbxproj	
@@ -9,6 +9,7 @@
 /* Begin PBXBuildFile section */
 		271631AE2EFF81FF00BC0309 /* GoogleSignIn in Frameworks */ = {isa = PBXBuildFile; productRef = 271631AD2EFF81FF00BC0309 /* GoogleSignIn */; };
 		271631B02EFF81FF00BC0309 /* GoogleSignInSwift in Frameworks */ = {isa = PBXBuildFile; productRef = 271631AF2EFF81FF00BC0309 /* GoogleSignInSwift */; };
+		276D2AF02F198BEB005D0306 /* StoreKit.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 276D2AEF2F198BEB005D0306 /* StoreKit.framework */; };
 		277F96BD2ED2A9CD002B7782 /* Auth in Frameworks */ = {isa = PBXBuildFile; productRef = 277F96BC2ED2A9CD002B7782 /* Auth */; };
 		277F96BF2ED2A9CD002B7782 /* PostgREST in Frameworks */ = {isa = PBXBuildFile; productRef = 277F96BE2ED2A9CD002B7782 /* PostgREST */; };
 		277F96C12ED2A9CD002B7782 /* Storage in Frameworks */ = {isa = PBXBuildFile; productRef = 277F96C02ED2A9CD002B7782 /* Storage */; };
@@ -34,7 +35,9 @@
 /* End PBXContainerItemProxy section */
 
 /* Begin PBXFileReference section */
-		277F96772ED2A6C7002B7782 /* Creator AI Studio.app */ = {isa = PBXFileReference; explicitFileType = wrapper.application; includeInIndex = 0; path = "Creator AI Studio.app"; sourceTree = BUILT_PRODUCTS_DIR; };
+		276D2AEE2F198AA1005D0306 /* Runspeed AI.entitlements */ = {isa = PBXFileReference; lastKnownFileType = text.plist.entitlements; path = "Runspeed AI.entitlements"; sourceTree = "<group>"; };
+		276D2AEF2F198BEB005D0306 /* StoreKit.framework */ = {isa = PBXFileReference; lastKnownFileType = wrapper.framework; name = StoreKit.framework; path = System/Library/Frameworks/StoreKit.framework; sourceTree = SDKROOT; };
+		277F96772ED2A6C7002B7782 /* Runspeed AI.app */ = {isa = PBXFileReference; explicitFileType = wrapper.application; includeInIndex = 0; path = "Runspeed AI.app"; sourceTree = BUILT_PRODUCTS_DIR; };
 		277F96872ED2A6CA002B7782 /* Creator AI StudioTests.xctest */ = {isa = PBXFileReference; explicitFileType = wrapper.cfbundle; includeInIndex = 0; path = "Creator AI StudioTests.xctest"; sourceTree = BUILT_PRODUCTS_DIR; };
 		277F96912ED2A6CA002B7782 /* Creator AI StudioUITests.xctest */ = {isa = PBXFileReference; explicitFileType = wrapper.cfbundle; includeInIndex = 0; path = "Creator AI StudioUITests.xctest"; sourceTree = BUILT_PRODUCTS_DIR; };
 /* End PBXFileReference section */
@@ -65,6 +68,7 @@
 				277F96C42ED2AA6C002B7782 /* Supabase in Frameworks */,
 				277F96C12ED2A9CD002B7782 /* Storage in Frameworks */,
 				271631B02EFF81FF00BC0309 /* GoogleSignInSwift in Frameworks */,
+				276D2AF02F198BEB005D0306 /* StoreKit.framework in Frameworks */,
 				277F96BF2ED2A9CD002B7782 /* PostgREST in Frameworks */,
 				277F96BD2ED2A9CD002B7782 /* Auth in Frameworks */,
 				27E2E0772ED51473004B79DA /* Kingfisher in Frameworks */,
@@ -92,6 +96,7 @@
 		277F966E2ED2A6C7002B7782 = {
 			isa = PBXGroup;
 			children = (
+				276D2AEE2F198AA1005D0306 /* Runspeed AI.entitlements */,
 				277F96792ED2A6C7002B7782 /* Creator AI Studio */,
 				277F968A2ED2A6CA002B7782 /* Creator AI StudioTests */,
 				277F96942ED2A6CA002B7782 /* Creator AI StudioUITests */,
@@ -103,7 +108,7 @@
 		277F96782ED2A6C7002B7782 /* Products */ = {
 			isa = PBXGroup;
 			children = (
-				277F96772ED2A6C7002B7782 /* Creator AI Studio.app */,
+				277F96772ED2A6C7002B7782 /* Runspeed AI.app */,
 				277F96872ED2A6CA002B7782 /* Creator AI StudioTests.xctest */,
 				277F96912ED2A6CA002B7782 /* Creator AI StudioUITests.xctest */,
 			);
@@ -113,6 +118,7 @@
 		277F96C22ED2AA6C002B7782 /* Frameworks */ = {
 			isa = PBXGroup;
 			children = (
+				276D2AEF2F198BEB005D0306 /* StoreKit.framework */,
 			);
 			name = Frameworks;
 			sourceTree = "<group>";
@@ -120,9 +126,9 @@
 /* End PBXGroup section */
 
 /* Begin PBXNativeTarget section */
-		277F96762ED2A6C7002B7782 /* Creator AI Studio */ = {
+		277F96762ED2A6C7002B7782 /* Runspeed AI */ = {
 			isa = PBXNativeTarget;
-			buildConfigurationList = 277F969B2ED2A6CA002B7782 /* Build configuration list for PBXNativeTarget "Creator AI Studio" */;
+			buildConfigurationList = 277F969B2ED2A6CA002B7782 /* Build configuration list for PBXNativeTarget "Runspeed AI" */;
 			buildPhases = (
 				277F96732ED2A6C7002B7782 /* Sources */,
 				277F96742ED2A6C7002B7782 /* Frameworks */,
@@ -135,7 +141,7 @@
 			fileSystemSynchronizedGroups = (
 				277F96792ED2A6C7002B7782 /* Creator AI Studio */,
 			);
-			name = "Creator AI Studio";
+			name = "Runspeed AI";
 			packageProductDependencies = (
 				277F96BC2ED2A9CD002B7782 /* Auth */,
 				277F96BE2ED2A9CD002B7782 /* PostgREST */,
@@ -146,7 +152,7 @@
 				271631AF2EFF81FF00BC0309 /* GoogleSignInSwift */,
 			);
 			productName = "Creator AI Studio";
-			productReference = 277F96772ED2A6C7002B7782 /* Creator AI Studio.app */;
+			productReference = 277F96772ED2A6C7002B7782 /* Runspeed AI.app */;
 			productType = "com.apple.product-type.application";
 		};
 		277F96862ED2A6CA002B7782 /* Creator AI StudioTests */ = {
@@ -237,7 +243,7 @@
 			projectDirPath = "";
 			projectRoot = "";
 			targets = (
-				277F96762ED2A6C7002B7782 /* Creator AI Studio */,
+				277F96762ED2A6C7002B7782 /* Runspeed AI */,
 				277F96862ED2A6CA002B7782 /* Creator AI StudioTests */,
 				277F96902ED2A6CA002B7782 /* Creator AI StudioUITests */,
 			);
@@ -295,12 +301,12 @@
 /* Begin PBXTargetDependency section */
 		277F96892ED2A6CA002B7782 /* PBXTargetDependency */ = {
 			isa = PBXTargetDependency;
-			target = 277F96762ED2A6C7002B7782 /* Creator AI Studio */;
+			target = 277F96762ED2A6C7002B7782 /* Runspeed AI */;
 			targetProxy = 277F96882ED2A6CA002B7782 /* PBXContainerItemProxy */;
 		};
 		277F96932ED2A6CA002B7782 /* PBXTargetDependency */ = {
 			isa = PBXTargetDependency;
-			target = 277F96762ED2A6C7002B7782 /* Creator AI Studio */;
+			target = 277F96762ED2A6C7002B7782 /* Runspeed AI */;
 			targetProxy = 277F96922ED2A6CA002B7782 /* PBXContainerItemProxy */;
 		};
 /* End PBXTargetDependency section */
@@ -430,10 +436,11 @@
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				ASSETCATALOG_COMPILER_GLOBAL_ACCENT_COLOR_NAME = AccentColor;
+				CODE_SIGN_ENTITLEMENTS = "Runspeed AI.entitlements";
 				CODE_SIGN_STYLE = Automatic;
 				CURRENT_PROJECT_VERSION = 1;
 				DEVELOPMENT_ASSET_PATHS = "\"Creator AI Studio/Preview Content\"";
-				DEVELOPMENT_TEAM = A2UK5VZ977;
+				DEVELOPMENT_TEAM = 3226HD5BV7;
 				ENABLE_PREVIEWS = YES;
 				GENERATE_INFOPLIST_FILE = YES;
 				INFOPLIST_FILE = "Creator-AI-Studio-Info.plist";
@@ -445,18 +452,21 @@
 				INFOPLIST_KEY_UIApplicationSceneManifest_Generation = YES;
 				INFOPLIST_KEY_UIApplicationSupportsIndirectInputEvents = YES;
 				INFOPLIST_KEY_UILaunchScreen_Generation = YES;
-				INFOPLIST_KEY_UISupportedInterfaceOrientations_iPad = "UIInterfaceOrientationPortrait UIInterfaceOrientationPortraitUpsideDown UIInterfaceOrientationLandscapeLeft UIInterfaceOrientationLandscapeRight";
-				INFOPLIST_KEY_UISupportedInterfaceOrientations_iPhone = "UIInterfaceOrientationPortrait UIInterfaceOrientationLandscapeLeft UIInterfaceOrientationLandscapeRight";
+				INFOPLIST_KEY_UISupportedInterfaceOrientations = UIInterfaceOrientationPortrait;
 				LD_RUNPATH_SEARCH_PATHS = (
 					"$(inherited)",
 					"@executable_path/Frameworks",
 				);
 				MARKETING_VERSION = 1.0;
-				PRODUCT_BUNDLE_IDENTIFIER = "com.michaelkogan.Creator-AI-Studio";
+				PRODUCT_BUNDLE_IDENTIFIER = com.michaelkogan.runspeedaiv1;
 				PRODUCT_NAME = "$(TARGET_NAME)";
+				SUPPORTED_PLATFORMS = "iphoneos iphonesimulator";
+				SUPPORTS_MACCATALYST = NO;
+				SUPPORTS_MAC_DESIGNED_FOR_IPHONE_IPAD = NO;
+				SUPPORTS_XR_DESIGNED_FOR_IPHONE_IPAD = NO;
 				SWIFT_EMIT_LOC_STRINGS = YES;
 				SWIFT_VERSION = 5.0;
-				TARGETED_DEVICE_FAMILY = "1,2";
+				TARGETED_DEVICE_FAMILY = 1;
 			};
 			name = Debug;
 		};
@@ -465,10 +475,11 @@
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				ASSETCATALOG_COMPILER_GLOBAL_ACCENT_COLOR_NAME = AccentColor;
+				CODE_SIGN_ENTITLEMENTS = "Runspeed AI.entitlements";
 				CODE_SIGN_STYLE = Automatic;
 				CURRENT_PROJECT_VERSION = 1;
 				DEVELOPMENT_ASSET_PATHS = "\"Creator AI Studio/Preview Content\"";
-				DEVELOPMENT_TEAM = A2UK5VZ977;
+				DEVELOPMENT_TEAM = 3226HD5BV7;
 				ENABLE_PREVIEWS = YES;
 				GENERATE_INFOPLIST_FILE = YES;
 				INFOPLIST_FILE = "Creator-AI-Studio-Info.plist";
@@ -480,18 +491,21 @@
 				INFOPLIST_KEY_UIApplicationSceneManifest_Generation = YES;
 				INFOPLIST_KEY_UIApplicationSupportsIndirectInputEvents = YES;
 				INFOPLIST_KEY_UILaunchScreen_Generation = YES;
-				INFOPLIST_KEY_UISupportedInterfaceOrientations_iPad = "UIInterfaceOrientationPortrait UIInterfaceOrientationPortraitUpsideDown UIInterfaceOrientationLandscapeLeft UIInterfaceOrientationLandscapeRight";
-				INFOPLIST_KEY_UISupportedInterfaceOrientations_iPhone = "UIInterfaceOrientationPortrait UIInterfaceOrientationLandscapeLeft UIInterfaceOrientationLandscapeRight";
+				INFOPLIST_KEY_UISupportedInterfaceOrientations = UIInterfaceOrientationPortrait;
 				LD_RUNPATH_SEARCH_PATHS = (
 					"$(inherited)",
 					"@executable_path/Frameworks",
 				);
 				MARKETING_VERSION = 1.0;
-				PRODUCT_BUNDLE_IDENTIFIER = "com.michaelkogan.Creator-AI-Studio";
+				PRODUCT_BUNDLE_IDENTIFIER = com.michaelkogan.runspeedaiv1;
 				PRODUCT_NAME = "$(TARGET_NAME)";
+				SUPPORTED_PLATFORMS = "iphoneos iphonesimulator";
+				SUPPORTS_MACCATALYST = NO;
+				SUPPORTS_MAC_DESIGNED_FOR_IPHONE_IPAD = NO;
+				SUPPORTS_XR_DESIGNED_FOR_IPHONE_IPAD = NO;
 				SWIFT_EMIT_LOC_STRINGS = YES;
 				SWIFT_VERSION = 5.0;
-				TARGETED_DEVICE_FAMILY = "1,2";
+				TARGETED_DEVICE_FAMILY = 1;
 			};
 			name = Release;
 		};
@@ -501,9 +515,9 @@
 				BUNDLE_LOADER = "$(TEST_HOST)";
 				CODE_SIGN_STYLE = Automatic;
 				CURRENT_PROJECT_VERSION = 1;
-				DEVELOPMENT_TEAM = A2UK5VZ977;
+				DEVELOPMENT_TEAM = 3226HD5BV7;
 				GENERATE_INFOPLIST_FILE = YES;
-				IPHONEOS_DEPLOYMENT_TARGET = 18.2;
+				IPHONEOS_DEPLOYMENT_TARGET = 17.6;
 				MARKETING_VERSION = 1.0;
 				PRODUCT_BUNDLE_IDENTIFIER = "com.michaelkogan.Creator-AI-StudioTests";
 				PRODUCT_NAME = "$(TARGET_NAME)";
@@ -520,9 +534,9 @@
 				BUNDLE_LOADER = "$(TEST_HOST)";
 				CODE_SIGN_STYLE = Automatic;
 				CURRENT_PROJECT_VERSION = 1;
-				DEVELOPMENT_TEAM = A2UK5VZ977;
+				DEVELOPMENT_TEAM = 3226HD5BV7;
 				GENERATE_INFOPLIST_FILE = YES;
-				IPHONEOS_DEPLOYMENT_TARGET = 18.2;
+				IPHONEOS_DEPLOYMENT_TARGET = 17.6;
 				MARKETING_VERSION = 1.0;
 				PRODUCT_BUNDLE_IDENTIFIER = "com.michaelkogan.Creator-AI-StudioTests";
 				PRODUCT_NAME = "$(TARGET_NAME)";
@@ -579,7 +593,7 @@
 			defaultConfigurationIsVisible = 0;
 			defaultConfigurationName = Release;
 		};
-		277F969B2ED2A6CA002B7782 /* Build configuration list for PBXNativeTarget "Creator AI Studio" */ = {
+		277F969B2ED2A6CA002B7782 /* Build configuration list for PBXNativeTarget "Runspeed AI" */ = {
 			isa = XCConfigurationList;
 			buildConfigurations = (
 				277F969C2ED2A6CA002B7782 /* Debug */,
diff --git a/Creator AI Studio.xcodeproj/xcuserdata/mike.xcuserdatad/xcschemes/xcschememanagement.plist b/Creator AI Studio.xcodeproj/xcuserdata/mike.xcuserdatad/xcschemes/xcschememanagement.plist
index 46371b0..e497cc2 100644
--- a/Creator AI Studio.xcodeproj/xcuserdata/mike.xcuserdatad/xcschemes/xcschememanagement.plist	
+++ b/Creator AI Studio.xcodeproj/xcuserdata/mike.xcuserdatad/xcschemes/xcschememanagement.plist	
@@ -9,6 +9,29 @@
 			<key>orderHint</key>
 			<integer>0</integer>
 		</dict>
+		<key>Promises (Playground).xcscheme</key>
+		<dict>
+			<key>orderHint</key>
+			<integer>1</integer>
+		</dict>
+	</dict>
+	<key>SuppressBuildableAutocreation</key>
+	<dict>
+		<key>277F96762ED2A6C7002B7782</key>
+		<dict>
+			<key>primary</key>
+			<true/>
+		</dict>
+		<key>277F96862ED2A6CA002B7782</key>
+		<dict>
+			<key>primary</key>
+			<true/>
+		</dict>
+		<key>277F96902ED2A6CA002B7782</key>
+		<dict>
+			<key>primary</key>
+			<true/>
+		</dict>
 	</dict>
 </dict>
 </plist>
diff --git a/Creator AI Studio/.DS_Store b/Creator AI Studio/.DS_Store
index 8c2892e..019c051 100644
Binary files a/Creator AI Studio/.DS_Store and b/Creator AI Studio/.DS_Store differ
diff --git a/Creator AI Studio/Pages/5 Profile/Settings.swift b/Creator AI Studio/Pages/5 Profile/Settings.swift
index 686aefd..f43f0f7 100644
--- a/Creator AI Studio/Pages/5 Profile/Settings.swift	
+++ b/Creator AI Studio/Pages/5 Profile/Settings.swift	
@@ -82,15 +82,11 @@ struct Settings: View {
                 }
             }
             
-<<<<<<< HEAD
-            // Purchase Credits section
-            Section("Credits") {
-                // Current Balance
-=======
+
             // Subscription section
             Section("Balance") {
                                // Current Balance
->>>>>>> eb4c2e7 (Add Google logo to sign-in buttons)
+
                 HStack {
                     Image(systemName: "dollarsign.circle.fill")
                         .foregroundColor(.green)
diff --git a/Runspeed AI.entitlements b/Runspeed AI.entitlements
index a812db5..80b5221 100644
--- a/Runspeed AI.entitlements	
+++ b/Runspeed AI.entitlements	
@@ -2,6 +2,8 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+	<key>aps-environment</key>
+	<string>development</string>
 	<key>com.apple.developer.applesignin</key>
 	<array>
 		<string>Default</string>
